src/AirtableClient.php

```diff
@@ -11,30 +13,51 @@ class AirtableClient
 {
     private $key;
     private $id;
-    
-    public function __construct($key, $id)
+    private $httpClient;
+    private $normalizer;
+
+    public function __construct($key, $id, HttpClientInterface $httpClient, ObjectNormalizer $objectNormalizer)
     {
         $this->key = $key;
         $this->id = $id;
+        $this->httpClient = $httpClient;
+        $this->normalizer = $objectNormalizer;
     }
-    
+
     /**
-     * findAll
+     * Returns a set of rows from AirTable
      *
-     * @param  mixed $table Table name
-     * @param  mixed $view  View name
+     * @param  mixed   $table Table name
+     * @param  mixed   $view  View name
+     * @param  string  $dataClass The class name which will hold fields data
      * @return array
      */
-    public function findAll(string $table, ?string $view = null): array
+    public function findAll(string $table, ?string $view = null, ?string $dataClass = null): array
```

Comment:
not related to your PR but you should rename the class `AirtableClient` to `AirtableRepository`